packages/fast-components-react-base/src/viewport-positioner/README.md

```diff
@@ -8,6 +8,8 @@ The *viewport positioner* and the anchor element it is attached to must both be
 
 If no "anchor" is specified the component will be disabled and not attempt to adjust layout. If no viewport is specified the component will assume the viewport is the root document element of the page.
 
+Authors should ensure that the positioner element has fixed dimensions, most likely coming from css styles, sizing forced by a parent element (i.e. grid cell), etc....  
```

Comment:
This sentence seems problematic, I would suggest:

"Authors should ensure that the positioner element has fixed dimensions. If these are dynamic this can cause incorrect positioning to be calculated."

Reply:
yes, better